packages/input-amount/src/LionInputAmount.js

```diff
@@ -127,4 +128,12 @@ export class LionInputAmount extends LocalizeMixin(LionInput) {
     // sufficient, it should also contain the abbreviation.
     this._currencyDisplayNode.setAttribute('aria-label', getCurrencyName(this.currency));
   }
+
+  __CurrencyLabel() {
```

Comment:
Maybe make this `get __currencyLabel() `?

Reply:
Thanks @tlouisse . Renamed the method name.